### PR TITLE
Leadfeeder new actions sources

### PR DESCRIPTION
**Pull request by jcortes**

### CODE REVIEW

File: components/leadfeeder/leadfeeder.app.mjs [hunk 1]
```diff
@@ -1,11 +1,146 @@
...
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    getUrl(path) {
+      return `${constants.BASE_URL}${path}`;
+    },
+    getHeaders(headers) {
+      return {
+        "Content-Type": "application/json",
+        "Authorization": `Token token=${this.$auth.api_token}`,
```
Comment (lcaresia) [inline components/leadfeeder/leadfeeder.app.mjs:56]: Move this to a getter method!

```suggestion
        "Authorization": `Token token=${this._apiToken()}`,
```

**Review by lcaresia — CHANGES_REQUESTED** [review]
Thanks for your contribution!

Need just one fix!

**Review by lcaresia — APPROVED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hello @jcortes, I've successfully installed the feedleader into a live wordpress website, though I couldn't create any new leads eventhough I've waited for a while




I think for now I'll let it ready for release, then we'll track for future issue if needed

**Pull request merged**